mri/operators/fourier/non_cartesian.py

```diff
@@ -567,11 +331,6 @@ def __init__(self, samples, shape, implementation='cpu', n_coils=1,
             self.density_comp = density_comp
```

Comment:
Kindly change the documentation also 
`implementation: str 'cpu' | 'cuda' | 'opencl' | 'gpuNUFFT',` We need to remove 'opencl' and 'cuda'